src/drivers/boards/px4fmu-v4/board_config.h

```diff
@@ -86,12 +86,17 @@
 
 #define GPIO_SPI_CS_FRAM		(GPIO_OUTPUT|GPIO_PUSHPULL|GPIO_SPEED_2MHz|GPIO_OUTPUT_SET|GPIO_PORTD|GPIO_PIN10)
 
+#define GPIO_SPI_CS_BMI055_ACC  (GPIO_OUTPUT|GPIO_PUSHPULL|GPIO_SPEED_2MHz|GPIO_OUTPUT_SET|GPIO_PORTC|GPIO_PIN15)
```

Comment:
If I follow the question. The logical PX4_SPIDEV_BMI055_ACC and say PX4_SPIDEV_MPU the will select the same GPIO. So it would be best in the board_config to rename the GPIO to something like GPIO_SPIb_CSn (where b is the buss and n is the CS Port number). Say GPIO_SPI0_CS1.

Then in the c code only touch GPIO_SPI0_CS1 and have the case have PX4_SPIDEV_BMI055_ACC, PX4_SPIDEV_MPU 

Then the whoami will sort things out.